mlflow/tracking/fluent.py

```diff
@@ -1238,13 +1243,14 @@ def delete_experiment(experiment_id: str) -> None:
         print("Name: {}".format(experiment.name))
         print("Artifact Location: {}".format(experiment.artifact_location))
         print("Lifecycle_stage: {}".format(experiment.lifecycle_stage))
-
+        print("Last Updated timestamp: {}".format(experiment.last_update_time))
     .. code-block:: text
         :caption: Output
 
         Name: New Experiment
         Artifact Location: file:///.../mlruns/2
         Lifecycle_stage: deleted
+        Last Updated timestamp: 1661624204
```

Comment:
```suggestion
        Last Updated timestamp: 1661624204
```

Is `1661624204` in milli seconds?

Reply:
no,  changed to millisecond